### ISSUE TITLE
Remove "global_step" in usage example of CosineDecay and other related classes

### ISSUE THREAD

**Issue by vsuryamurthy** [tensorflow/tensorflow#30437]
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:

https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/keras/optimizer_v2/learning_rate_schedule.py

## Description of issue (what needs changing):
Usage example of CosineDecay, CosineDecayRestarts, LinearCosineDecay, NoisyLinearCosineDecay takes a parameter "global_step" which does not correspond to the definition. 

### Clear description

The script defines the various learning rate decay classes used for network training.

### Usage example

There is a usage example. However, it does not correspond to the definition. 

### Submit a pull request?

I can if required

**Comment by jvishnuvardhan** [comment]
@vsuryamurthy Can you submit a PR to update the docs? Thanks!

**Comment by jvishnuvardhan** [comment]
Closing this issue as it was resolved by the  https://github.com/tensorflow/tensorflow/pull/30648 which was already merged. Thanks!
